#! /bin/sh -e
## 10_emacs_regex.dpatch by Andreas Metzler
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Revert back too using emacs-style regex by default.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
	-patch) patch -f --no-backup-if-mismatch -p1 < $0;;
	-unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
@DPATCH@
diff -ur findutils-4.2.22.orig/find/parser.c findutils-4.2.22/find/parser.c
--- findutils-4.2.22.orig/find/parser.c	2005-06-08 00:18:41.000000000 +0200
+++ findutils-4.2.22/find/parser.c	2005-07-04 12:05:42.939878472 +0200
@@ -1392,7 +1392,7 @@
   struct predicate *our_pred;
   struct re_pattern_buffer *re;
   const char *error_message;
-  int opt = RE_SYNTAX_POSIX_BASIC;
+  int opt = RE_SYNTAX_EMACS;
 
   if ((argv == NULL) || (argv[*arg_ptr] == NULL))
     return (false);
